Crawl/Crawl/Crawl.cs

```diff
@@ -116,8 +116,39 @@ public class Crawl
                 }
                 catch (WebException we)
                 {
-                    if ((we.Response as HttpWebResponse)?.StatusCode == HttpStatusCode.Forbidden)
-                        continue;
+                    HttpWebResponse httpResponse = we.Response as HttpWebResponse;
+                    if (we.Status == WebExceptionStatus.ServerProtocolViolation)
+                    {
+                        // Get a little more telemetry about what is going on here.
+                        IDictionary<string, string> traceData = new Dictionary<string, string>();
+                        traceData["Response.SupportsHeaders"] = we.Response.SupportsHeaders.ToString();
```

Comment:
why not use Dictionary initializer?

Reply:
No reason - artifact of my having added "SupportsHeaders" as an after-thought. I'll change it back.